sdc/tests/test_dataframe.py

```diff
@@ -1879,6 +1879,15 @@ def test_impl():
         self.assertTrue(isinstance(two, np.ndarray))
         self.assertTrue(isinstance(three, np.ndarray))
 
+    def test_df_len(self):
+        def test_impl(df):
+            return len(df)
+
+        hpat_func = self.jit(test_impl)
+        n = 11
+        df = pd.DataFrame({'A': np.arange(n), 'B': np.arange(n) ** 2})
+        assert (hpat_func(df), test_impl(df))
```

Comment:
This assert is always true.
Why do you use plain assert at all?

Reply:
because result len() - number.

Reply:
https://docs.python.org/3/library/unittest.html#assert-methods